types/rn-material-ui-textfield/rn-material-ui-textfield-tests.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, ReactElement, createRef } from 'react';
```

Comment:
Do you need this `ReactElement` import? It seems that this is not being used

Reply:
oh that was probably there from some testing
```suggestion
import { Component, createRef } from 'react';
```